back/src/bsvhu/resolvers/BsvhuMetadata.ts

```diff
@@ -61,10 +62,10 @@ const bsvhuMetadataResolvers: BsvhuMetadataResolvers = {
         );
         return [];
       } catch (errors) {
-        return errors.inner?.map(e => {
+        return errors.inner?.map((e: ValidationError) => {
```

Comment:
Pas lié au problème BSDA, mais necessaire pour debugger un autre bug en prod
https://sentry.incubateur.net/organizations/betagouv/issues/71426/?end=2023-11-06T23%3A59%3A59&project=19&query=is%3Aunresolved+&referrer=issue-stream&sort=new&start=2023-11-06T00%3A00%3A00&utc=true